src/views/OneTimeCode/index.tsx

```diff
@@ -79,7 +80,7 @@ const Authentication = () => {
         style={{ bottom: insets.bottom + 15 }}
         disabled={!!timer}
         onPress={() => {
-          setTimer(50);
+          setTimer(RESEND_TIMEOUT_IN_SECONDS);
```

Comment:
Great job with the PR removing magic numbers. Wouldn't it be nice to reuse the `INITIAL_TIMEOUT_IN_SECONDS` here instead of relying on multiple sources of truth? As the wanted behavior, in this example app, is just to reset it to the default on resend.

Reply:
I thought about adding this `RESEND_TIMEOUT_IN_SECONDS` , because I believe it would help to implement an anti-spam logic in the future.
I deeply appreciate your reply.

Reply:
Sure, it makes sense for a complete app! It was more of a YAGNI concern as it is just an example. Merging.